bin/pest

```diff
@@ -51,5 +53,11 @@ use Symfony\Component\Console\Output\OutputInterface;
         }
     }
 
+    $_SERVER['argv'] = InteractsWithPlugins::handleArguments($_SERVER['argv']);
+
+    if ($argv->hasParameterOption('--parallel')) {
+        exit(ParaTestCommand::applicationFactory($testSuite->rootPath)->run(new ArgvInput()));
```

Comment:
Refactored:

1) Plugin now has its own `Command` class, which is called instead of Paratest
2) We no longer morph the `$_SERVER['argv']` input at all, but rather transform an array